cylc/flow/cfgspec/globalcfg.py

```diff
@@ -283,6 +283,14 @@
                 file size.
             ''')
 
+    with Conf('install'):
+        Conf('source dirs', VDR.V_STRING_LIST, default=['~/cylc-src'], desc='''
+            A list of paths where workflow source directories may be found.
```

Comment:
I think I want to say why first, then what second, but you may disagree - my suggestion:

A list of paths where ``cylc install`` will look for source directories.
If a workflow source is in one of these directories you can install that workflow with ``cylc install REG``. Cylc UI uses source folders when searching for non-installed workflows.

Reply:
How about
```rst
A list of paths where ``cylc install <flow_name>`` will look for
a workflow of that name. All workflow source directories that are 
in these locations will show up in the UI, ready for installation.
```

Reply:
I should probably add
```rst
.. note:
   If workflow source directories of the same name exist in more
   than one of these paths, only the first one will be picked up.
```